impl_core/src/test/java/io/opencensus/implcore/trace/SpanBuilderImplTest.java

```diff
@@ -251,6 +251,64 @@ public void startChildSpan_WithoutSpecifiedSampler() {
     assertThat(childSpan.getContext().getTraceOptions().isSampled()).isFalse();
   }
 
+  @Test
+  public void startRemoteChildSpan_WithProbabilitySamplerDefaultSampler() {
+    when(traceConfig.getActiveTraceParams()).thenReturn(TraceParams.DEFAULT);
+    // This traceId will not be sampled by the ProbabilitySampler
```

Comment:
Same here, it would help to have an explanation of why this TraceId isn't sampled.

Reply:
done.